include/moveitem.h

```diff
@@ -297,3 +297,44 @@ int FreeSlotForItem(ItemClient* pItem)
 	}
 	return 0;
 }
+
+CItemLocation* FreeItemLocationForItem(CItemLocation* pFreeSlot, ItemClient* pItemClient)
+{
+	unsigned char  ucCurSize = 10;
+	ItemDefinition* pUnequipInfo = pItemClient->GetItemDefinition();
+	if (!pUnequipInfo) {
+		return nullptr;
+	}
+
+	for (unsigned short usSlot = InvSlot_FirstBagSlot; usSlot <= GetHighestAvailableBagSlot(); usSlot++)
+	{
+		if (ItemClient* pInvSlot = GetPcProfile()->GetInventorySlot(usSlot))
+		{
+			ItemDefinition* pItemInfo = pInvSlot->GetItemDefinition();
+
+			if (pItemInfo
+				&& pItemInfo->Combine != 2
```

Comment:
Use `ContainerType` instead of `Combine`.

2 maps to ContainerType_Quiver, so use that instead of 2.